### DISCUSSION TITLE
Missing doc for menu_item

### DISCUSSION THREAD

**Discussion by ed2050** [zauberzeug/nicegui#4107] [Ideas / Feature Requests]
AFAICT there's no documentation for `ui.menu_item ()`.  It's used on [ui.menu page](https://nicegui.io/documentation/menu) but no signature or args are given.  Did I miss it somewhere else?

**Comment by falkoschindler** [comment]
Thanks for spotting this, @ed2050!
I created PR #4115 to fix it.
